spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/config/StandardizationConfigParser.scala

```diff
@@ -129,20 +133,28 @@ object StandardizationConfigParser {
         config.withRawPathOverride(Some(value)))
         .text("override the path of the raw data (used internally for performance tests)"),
 
+      opt[Boolean]("empty-values-as-nulls").optional()
+        .action((value, config) => config.withFixedWidthTreatEmptyValuesAsNulls(Some(value)))
+        .text("For FixedWidth file format. Treats empty values as null. Default is false"),
```

Comment:
The only unclear thing to me is where the defaults are coming from.

Reply:
Does it make sense to just have it as Boolean instead of Option[Boolean]?

Reply:
The defaults are defaults of the FixedWidth file format. Same as CSV Delimiter, header, etc.